check/mode-lowmem.c

```diff
@@ -3351,6 +3351,31 @@ static int repair_extent_data_item(struct btrfs_root *root,
 	return err;
 }
 
+static int grab_inode_item(struct btrfs_root *root,
```

Comment:
The name `grab` is usually used for something regarding references when reading an object, please rename it to `read_inode_item`.